rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -365,22 +361,15 @@ def _add_blockchain_accounts_test_start(
 
         if async_query:
             task_id = assert_ok_async_response(response)
-            result = wait_for_async_task_with_result(
+            wait_for_async_task(
                 api_server,
                 task_id,
                 timeout=ASYNC_TASK_WAIT_TIMEOUT * 4,
             )
         else:
-            result = assert_proper_response_with_result(response)
+            assert_proper_response(response)
 
-    assert_eth_balances_result(
```

Comment:
In this file and generally in **all** tests you are removing a lot of testing functionality.

We were using these tests to also test the balance querying functionality I think. If yes you need to keep testing it. Just make explicit balance queries.

If there is other tests that already test balance queries of this particular chain it's fine to not do it though.